### ISSUE TITLE
GNUWin

### ISSUE THREAD

**Issue by Prytanis** [hist3907b-winter2015/module2-findingdata#3]
The install for CoreUtils had testing instructions in them involving the windows CMD prompt.  When executing the test, it did not function.  In viewing what documentation that I could find, Windows 8, which is what my machine runs, does not seem to be supported.

All subsequent parts of the Module have failed for me.  This may not be my only issue, but it is the only one that I was savvy enough to detect.

Can you confirm that Windows 8 is not supported?  If so, is there a work around?

**Comment by shawngraham** [comment]
hmmm. windows 8 no go, eh? damn. alright, we'll just have to figure out an alternative. thanks for trying that.

**Comment by AllisonKuhl** [comment]
yeah I don't think coreutils worked for me either. I also have an older version of windows.

**Comment by shawngraham** [comment]
Allison, your last email to me indicated that everything worked for you. For you, see the earlier issue: you need to be putting the file path to the coreutils folder into the PATH variable - there were screenshots. And remember that git shell is NOT git bash.

**Comment by shawngraham** [comment]
Anyway, this is one of the things I wanted to work with you all tomorrow in class.

**Comment by shawngraham** [comment]
For Windows 8 (I'm not certain this works, so treat this as a memo until I can find a windows 8 machine to test it; following [this](http://outcoldman.com/en/archive/2014/07/20/scoop/) ):
- don't worry about git bash
- find powershell on your machine (search your progams), start it up.
- type in the following commands. What we're doing here is getting the tools we need added to your powershell.
- `$ set-executionpolicy unrestricted -s cu`
- `$ iex (new-object net.webclient).downloadstring('https://get.scoop.sh')`
- `scoop install 7zip coreutils curl git grep openssh sed wget vim grep`
- now, at the powershell prompt change directory to wherever you were working on this exercise (where you downloaded jq, and where you saved the api example sh program). Try running the command.

As I say, I don't have access to a Windows 8 machine at the moment, but this might work.

**Comment by AllisonKuhl** [comment]
For me, I copied the seq.exe into the file where the program is and tried the program and it seemed to work. Or at least in the result file I got a bunch of oochim numbers.

**Comment by shawngraham** [comment]
that also works. it's better to add the coreutils directory to your PATH variable; otherwise, you'd have to copy the command (in this case, 'seq.exe') into where you were working. But getting the oochim numbers: yep, that's what you want. The final line of the program passes all those numbers to wget, and starts downloading...

**Comment by shawngraham** [comment]
UPDATE:
For Windows 8 - the instructions in my note above do work. Open up powershell, run those three commands. (Say 'yes' at the prompt after the first instruction). Once the smoke clears, navigate to wherever you've got the program for exercise 3 located (eg. `cd users\user\documents\exercises` on my machine). 

To run the program, type `./api-ex-pc.sh`  and soon you'll be downloading materials from Canadiana.
